doc/quickstart/install.rst

```diff
@@ -208,3 +208,40 @@ yourself if you are just running diagnostics:
  - DKRZ-Mistral: `esmvaltool` is available on login nodes (`mistral.dkrz.de`) and pre- and post-processing
    nodes (`mistralpp.dkrz.de`) after login and module loading via `module load esmvaltool`; the command
    `module help esmvaltool` provides some information about the module.
+
+Installation from the conda lock file
+-------------------------------------
+
+A fast conda environment creation is possible using the provided conda lock files.
```

Comment:
Do I need to create a conda lock file to use it for installation purposes? I think not, so maybe it would make more sense to start with explaining how you install from the conda lock file and then afterwards explain how to create your own if you want to?

Reply:
cheers @bvreede - good call! Lemme write it that way then :+1: